posix.mak

```diff
@@ -183,12 +183,12 @@ ${LATEST}.ddoc :
 	echo "LATEST=${LATEST}" >$@
 
 clean:
-	rm -rf $(DOC_OUTPUT_DIR) ${LATEST}.ddoc
+	rm -rf $(DOC_OUTPUT_DIR) ${LATEST}.ddoc dpl-docs/.dub
```

Comment:
It's the cached binaries. You normaly don't want to delete that. Dub will rebuild the tool if necessary.